package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@digicatapult/check-version",
-  "version": "1.1.42",
+  "version": "1.1.43",
```

Comment:
As we are changing the behaviour in a non-backwards compatible way, this should be a major version bump.

Reply:
ok